snuba/query/processors/uuid_column_processor.py

```diff
@@ -139,6 +145,7 @@ def process_condition(self, exp: Expression) -> Expression:
                     new_params.append(column)
 
             left_exp, right_exp = new_params
+            metrics.increment("query_processed", tags={"type": "bare"})
```

Comment:
Up to you but not sure what you are trying to count. It seems that "formatted" counts the number of `IN` optimized clauses while "bare" means the number of optimized individual conditions. If that's intended, I think a better naming should be advisable. 
Also this will count the number of conditions, not the number of queries. Is this intended?

Reply:
I thought I would count how often the UUID column is wrapped in the replace function vs not wrapped. It's not really that important.

I did change it to only increment once per query, instead of once per expression transformed.